modules/end-user-guide/partials/proc_creating-a-workspace-from-the-default-branch-of-a-git-repository.adoc

```diff
@@ -22,5 +22,5 @@ Run the workspace by opening the following URL: `pass:c,a,q[{prod-url}/#https://
 .Example
 [subs="+quotes"]
 ----
-https://che.openshift.io/#https://github.com/eclipse/che
+https://workspaces.openshift.com/#https://github.com/eclipse/che
```

Comment:
@themr0c this is an incorrect reference to factory - should be based on `f?url= `see #1884